services/vision/client.go

```
@@ -242,6 +242,26 @@ func protoToObjects(pco []*commonpb.PointCloudObject) ([]*vision.Object, error)
 	return objects, nil
 }
 
+func (c *client) GetProperties(ctx context.Context, extra map[string]interface{}) *Properties {
```

The method should also return an error